source/deploy/standalone/dev_ssl.md.erb

```diff
@@ -0,0 +1,845 @@
+---
+title: Using SSL with Passenger in development
+section: walkthroughs
```

Comment:
The file is under deploy, not a walkthrough. Can you also commit the changes to the deployment index?

Furthermore, it looks like you're providing separate instructions for nginx / apache / standalone, you should add a chooser so that irrelevant info can be filtered out.